ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/valueprovider/PrefixUtilTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.valueprovider;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class PrefixUtilTest {
+
+  @ParameterizedTest
+  @ValueSource(strings = {"--test", "-test", "test"})
+  void optionIsStrippedOfDashPrefix(final String option) {
+    final String stripped = PrefixUtil.stripPrefix(option);
```

Comment:
should stripPrefix _take_ the prefix which is being removed? Or is it implicit? If its implicit - we should probably name it as such.